client/my-sites/sidebar/utils.js

```diff
@@ -71,5 +79,13 @@ export const itemLinkMatches = ( path, currentPath ) => {
 		return isManageAllSitesPluginsPath( path );
 	}
 
+	if ( pathIncludes( path, 'stats', 1 ) ) {
+		// For /store/stats and /google-my-business/stats paths.
+		return (
+			fragmentIsEqual( path, currentPath, 1 ) ||
+			fragmentIsEqualDifferentPositions( path, 1, currentPath, 2 )
+		);
+	}
```

Comment:
Out of curiosity: Is it possible to have a false positive using the new `fragmentIsEqualDifferentPositions` function? Would such a false positive would be limited to the my-sites path?

Reply:
Hi @jsnmoon That's true 😄 

`my-sites` doesn't have any URL pattern, does it? The safest way is to only test the store and gmb cases.

Reply:
Aha, good point! `my-sites` isn't restricted to a URL pattern. I agree that explicitly checking for store and GMB would be a safer implementation here 🙂

Reply:
I'll propose another PR with the approach. Thanks for pointing out.